mono/metadata/sgen-tarjan-bridge.c

```diff
@@ -517,6 +551,10 @@ push_object (GCObject *obj)
 #if DUMP_GRAPH
 		printf ("already marked\n");
 #endif
+
+		// Notice we only follow forward for non-initial objects (since forwarding is added on first scan)
```

Comment:
initial objects as in objects in state INITIAL. Would it be clearer if INITIAL were in all caps?